_sass/header.scss

```diff
@@ -0,0 +1,84 @@
+header {
+    background-color: #036;
+    border-bottom: 0.125em solid #fcba19;
+    padding: 0 1.875em 0 1.875em;
+    color: #fff;
+    display: flex;
+    height: 4.063em;
+    top: 0;
+    position: fixed;
+    width: 100%;
+    -webkit-box-shadow: 0 0.375em 0.500em -0.250em #b3b1b3;
+    -moz-box-shadow: 0 0.375em 0.500em -0.250em #b3b1b3;
+    box-shadow: 0 0.375em 0.500em -0.250em #b3b1b3;
+    z-index: 999;
+    h1 {
+        font-family: ‘BCSans’, ‘Noto Sans’, Verdana, Arial, sans-serif;
+        font-weight: normal;
+        margin: 0.313em 0.313em 0 1.125em;
+    }
+    .banner {
+        display: flex;
+        justify-content: flex-start;
+        align-items: center;
+        margin: 0 0.625em 0 0;
+    }
+    .other {
+        display: flex;
+        align-items: center;
+        flex-grow: 1;
+    }
+    img {
+        max-width: 100%;
+        margin-left: 0;
+        margin-right: 0;
+        margin-top: 0;
+        margin-bottom: 1.25rem;
+        padding-bottom: 0;
+        padding-left: 0;
+        padding-right: 0;
+        padding-top: 0;
```

Comment:
Can use the margin and padding shorthands here
https://alligator.io/css/padding-margin-shorthands/

Reply:
pushing commit up.  surprised that sass doesn't "fix" that...